trustmanager/x509utils.go

```diff
@@ -440,6 +451,11 @@ func EncryptPrivateKey(key data.PrivateKey, passphrase string) ([]byte, error) {
 		return nil, err
 	}
 
+	if encryptedPEMBlock.Headers == nil {
```

Comment:
I don't think this condition should ever happen, and if it does, we should error out, because clearly the key has not been correctly encrypted.